tests/model_indexes/tests.py

```diff
@@ -88,3 +89,36 @@ def test_abstract_children(self):
         self.assertEqual(index_names, ['model_index_name_440998_idx'])
         index_names = [index.name for index in ChildModel2._meta.indexes]
         self.assertEqual(index_names, ['model_index_name_b6c374_idx'])
+
+    @skipUnlessDBFeature('supports_tablespaces')
+    def test_db_tablespace(self):
+        with connection.schema_editor() as editor:
+            # Index with db_tablespace attribute.
+            for fields in [
+                # Field with db_tablespace specified on model.
+                ['shortcut'],
+                # Field without db_tablespace specified on model.
+                ['author'],
+                # Multi-column with db_tablespaces specified on model.
+                ['shortcut', 'isbn'],
+                # Multi-column without db_tablespace specified on model.
+                ['title', 'author'],
+            ]:
+                with self.subTest(fields=fields):
+                    index = models.Index(fields=fields, db_tablespace='idx_tbls2')
+                    self.assertIn('"idx_tbls2"', index.create_sql(Book, editor).lower())
+            # Indexes without db_tablespace attribute.
+            for fields in [['author'], ['shortcut', 'isbn'], ['title', 'author']]:
+                with self.subTest(fields=fields):
+                    index = models.Index(fields=fields)
+                    if settings.DEFAULT_INDEX_TABLESPACE:
```

Comment:
I don't understand why you need to check for this setting here. AFAICT we're not altering the setting during test runs, do we? Should the `else` clause be sufficient?